extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -237,6 +237,7 @@ angular.module('oppia').component('svgFilenameEditor', {
       };
 
       ctrl.postSvgToServer = function(dimensions, resampledFile) {
+        ctrl.isLoadingIndicatorShown = true;
```

Comment:
Let's move this to before the `postSvgToServer` method call (inside the `saveSvgFile` method), since the "false" counterpart of this statement is present there. It will make things easier to follow.

Reply:
Done